src/AI/JakeMan.java

```diff
@@ -74,8 +75,13 @@ public JakeMan(Army army)
     // move your leftover vehicles so they cover the tiles attacking most relevant stuff (contested cities, own units)
     // move infs as far towards nearest contested stuff as you can without underdefending, cap if possible
 
+    // Comment from L&F/Rize, on the idea of engaging e.g. a Lin COU on a city:
+    // IF the dude is free & there's multiple attackers & you can do multiple attacks (since having 2 tanks for 1 tile won't help),
+    //   check the total calc after shooting with every firepower thing that can shoot and not the normal one
```

Comment:
Is this English?

Reply:
Yes?
On the subject of attacking a unit that would be unappealing to hit with one unit (city tank with +stats)...
If I think I have local force superiority, and I can hit with multiple units, evaluate the entire sequence of shots rather than just a single attack.
(since the more hits you can do, the more favorable the overall outcome is liable to be)

Reply:
> "check the total calc after shooting with every firepower thing that can shoot and not the normal one"

Aside from the generally atrocious verbiage, it isn't even clear. Which one is the normal one?

Reply:
The normal calc to see if an attack is a good idea.
I don't recall if he ever laid that out precisely, but I came up with something decent, I think.

Reply:
I have a mild preference for well-formed English, and a strong preference for unambiguous prose in any case (and I may note these two often correlate). At minimum I'd at least like an antecedent for any pronouns in use (and you'll say there was one--"calc", presumably--but then we are back to the "unambiguous" requirement).

I'm admittedly impressed that you managed to derive an implementable requirement from what he wrote; I don't think I could have.

Anyway, guess I'll climb back down off this high horse of mine (until next time!)

Reply:
Very fair. It's also kind of plucked out of the larger context of the discussion, since "a chatlog" is questionable requirement text in its own ways.

Hah, thanks.

Reply:
Groovy.